test/e2e/specs/site-editor/new-templates-list.spec.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+async function getRows( page ) {
+	return page.locator( 'table.dataviews-list-view tbody tr' );
+}
+
+async function getTemplateTitles( page ) {
+	const rows = await ( await getRows( page ) ).all();
+	return Promise.all(
+		rows.map( ( row ) => row.locator( 'td' ).nth( 0 ).textContent() )
+	);
+}
+
+async function goToTemplatesList( admin, page ) {
+	await admin.visitSiteEditor();
+	await page.getByRole( 'button', { name: 'Templates' } ).click();
+	await page.getByRole( 'button', { name: 'Manage all templates' } ).click();
+}
+
+async function createTemplate( admin, page ) {
+	await admin.visitSiteEditor();
+	await page.getByRole( 'button', { name: 'Templates' } ).click();
+	await page.getByRole( 'button', { name: 'Add New Template' } ).click();
+	await page.locator( 'span', { hasText: 'Date Archives' } ).first().click();
+	// Wait a bit for the template to be created.
+	await page.getByRole( 'button', { name: 'Close' } ).click();
+}
+
+test.describe( 'Templates', () => {
```

Comment:
Do we expect this test to replace an existing test once we remove the flag? Does it cover the same things? more?

Reply:
I din't find any tests about the templates list, so I don't think we will remove something after making this stable(unless I missed something).